GuardiansoftheSpectrum/app/src/main/res/layout/activity_myinfo.xml

```diff
@@ -7,6 +7,17 @@
                 android:layout_height="match_parent"
                 android:weightSum="1">
 
+    <!-- <include layout="@layout/toolbar" /> -->
```

Comment:
Android Studio says that Relative Layout doesn't have include options. Leaving this one for the case if we change the layout.